rxbinding/src/main/java/com/jakewharton/rxbinding2/internal/Preconditions.java

```diff
@@ -27,6 +30,15 @@ public static void checkArgument(boolean assertion, String message) {
     return value;
   }
 
+  public static boolean isNotOnMainThread(Observer<?> observer) {
```

Comment:
I don't like this name. I was surprised `isSomething` method not only checks condition but also emit error to observer.